Packs/QRadar/ReleaseNotes/2_0_18.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
+- Enhanced the closing reason message of a mirrored incident when being closed.
```

Comment:
Seems kind of vague - lets try and make it clearer since it is mirroring related with some guideline questions:
1 - Which side is closing and which is being closed?
2 - Is this relevant to all mirroring incidents or just some?
3 - I think "closing reason" is an incident field no? if so it should be written like in the system and covered with **on each side**.

Reply:
1) in README it says, QRadar supports only mirror in (QRadar to XSOAR), so I thought it was clear.
2) All mirroring incidents
3) I talked about the XSOAR incident closing reason (as it is being printed when closing investigation).

Reply:
ok cool so lets just be explicit about this:
```
- Improved the outputted closing reason message when a mirrored incident is being closed (from QRadar to XSOAR).
```

Reply:
Adding docs approved and doing your proposed suggestion